app/Http/ViewHelpers/Company/CompanyQuestionViewHelper.php

```diff
@@ -35,10 +35,6 @@ public static function questions(Company $company): ?Collection
         foreach ($questions as $question) {
             $numberOfAnswers = $question->answers->count();
 
-            if ($numberOfAnswers == 0) {
```

Comment:
Not sure about deleting this part.
In this case we would also display questions that don't have any answers.

Reply:
I've move that to #765